### PR TITLE
Feature/storybook setup

### PR DESCRIPTION
**Pull request by deepakMc**
StoryBook - Setup completed and Test a component PageTitle.js

### CODE REVIEW

File: src/stories/1-Button.stories.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+export const Text = () => <Button onClick={action('clicked')}>Hello Button</Button>;
+
+export const Emoji = () => (
+  <Button onClick={action('clicked')}>
+    <span role="img" aria-label="so cool">
+      😀 😎 👍 💯
+    </span>
+  </Button>
+);
```
Comment (MDHayes) [inline src/stories/1-Button.stories.js]: We can delete this file, its just the demo story

File: src/stories/0-Welcome.stories.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+// import { linkTo } from '@storybook/addon-links';
+// import { Welcome } from '@storybook/react/demo';
+import PageTitle from '../components/PageTitle';
+
+export default {
+  title: 'Welcome',
```
Comment (MDHayes) [inline src/stories/0-Welcome.stories.js]: Can we change this name and the file name away from the example and reflect the Story here?

File: src/stories/0-Welcome.stories.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+// import { linkTo } from '@storybook/addon-links';
+// import { Welcome } from '@storybook/react/demo';
```
Comment (MDHayes) [inline src/stories/0-Welcome.stories.js]: Can we remove these imports?

**Review by MDHayes — CHANGES_REQUESTED** [review]
Good work 👍 Just a couple of small things it would be good to have changed.

File: src/stories/0-Welcome.stories.js (hunk 3 above)
Comment (deepakMc) [reply src/stories/0-Welcome.stories.js]: ok

File: src/stories/0-Welcome.stories.js (hunk 2 above)
Comment (deepakMc) [reply src/stories/0-Welcome.stories.js]: Need to change "title: 'Welcome'" as well?

File: src/stories/1-Button.stories.js (hunk 1 above)
Comment (deepakMc) [reply src/stories/1-Button.stories.js]: ok I will delete it

**Comment by deepakMc** [comment]
Please review updated PR

File: src/stories/0-Welcome.stories.js (hunk 2 above)
Comment (MDHayes) [reply src/stories/0-Welcome.stories.js]: It should have the component name so that its easy to navigate as we add more stories.

**Review by MDHayes — CHANGES_REQUESTED** [review]
Just the one change still on the name of the story.

File: src/stories/0-Welcome.stories.js (hunk 2 above)
Comment (deepakMc) [reply src/stories/0-Welcome.stories.js]: ok thanks

**Comment by deepakMc** [comment]
Please review

File: src/stories/0-Welcome.stories.js (hunk 2 above)
Comment (deepakMc) [reply src/stories/0-Welcome.stories.js]: changed the title with component name

**Review by chrissloey — APPROVED** [review]
Looks good

**Pull request merged**